libs/langchain/langchain/document_loaders/readthedocs.py

```diff
@@ -1,9 +1,13 @@
 from pathlib import Path
-from typing import Any, List, Optional, Sequence, Tuple, Union
+from typing import TYPE_CHECKING, Any, List, Optional, Sequence, Tuple, Union
 
 from langchain.docstore.document import Document
 from langchain.document_loaders.base import BaseLoader
 
+if TYPE_CHECKING:
+    from bs4 import NavigableString
+    from bs4.element import Comment, Tag
```

Comment:
@baskaryan added type checking, a lot of my parameters are bs4 objects, but truthfully couldn't find any examples within the codebase on how parameter type checking for bs4 objects are done. Would love to get your eyes on how I'm handling type checking for Tag, Comment, and NavigableString